hdmi/interfaces/aux_interface.py

```diff
@@ -6,12 +6,13 @@ class AuxInterface:
     def __init__(self, clock, aux_depth=(4, 4, 4)):
 
         """
+
          This interface is the internal interface modeled after
          the xapp495 internal audio interface
 
          Args:
-             :param clock: pixel clock.
-             :param aux_depth (optional): The bus width of the aux interface
+             clock: pixel clock.
```

Comment:
Use the Google style doc strings as described in the [best practices](http://discourse.myhdl.org/t/best-practices-for-cores-development-gsoc-2016-projects/30) (see the Documentation section).